python/ray/serve/_private/constants.py

```diff
@@ -168,6 +168,10 @@
 # future. Use RAY_SERVE_LOG_ENCODING or 'LoggingConfig' to enable json format.
 RAY_SERVE_ENABLE_JSON_LOGGING = os.environ.get("RAY_SERVE_ENABLE_JSON_LOGGING") == "1"
 
+# Setting RAY_SERVE_LOG_TO_STDERR=0 will disable logging to the stdout and stderr.
+# Also, redirect them to serve's log files.
+RAY_SERVE_LOG_TO_STDERR = not os.environ.get("RAY_SERVE_LOG_TO_STDERR") == "0"
```

Comment:
Let's follow the same pattern as the other env vars for this please
```suggestion
RAY_SERVE_LOG_TO_STDERR = (
    os.environ.get("RAY_SERVE_LOG_TO_STDERR", "1") == "1"
)
```

Reply:
I think the meaning is slightly different that's why I wrote it the way it is😅 If the user set the new env to anything not 1, its gonna disable stderr. Are we ok with it?

Reply:
Yea should be fine. We have other flags structured this way (e.g., `RAY_SERVE_EAGERLY_START_REPLACEMENT_REPLICAS`, `RAY_SERVE_ALWAYS_RUN_PROXY_ON_HEAD_NODE`, `RAY_SERVE_ENABLE_QUEUE_LENGTH_CACHE`).

Better to keep this behavior consistent than worry about unimportant edge cases